### DISCUSSION TITLE
Web app DNS Issue?

### DISCUSSION THREAD

**Discussion by stefanstranger** [vercel/community#6185] [Bug Report]
### Page to Investigate

https://azure-subnet-copilot.vercel.app/

### Steps to Reproduce

I cannot access my Web app hosted on Vercel.

Error message: Hmmm… can't reach this page.

Is there an dns issue?

**Comment by rsalzer** [comment]
I have the same problem with a custom domain purchased through Vercel (and I have not changed the default settings...) since yesterday.

**Reply by stefanstranger** [reply]
yes this has started since yesterday. Even tried a redeployment but that didn't resolve the issue.

**Reply by stefanstranger** [reply]
It's resolved.

**Comment by amyegan** [comment]
It looks like this was related to https://www.vercel-status.com/incidents/p77z6nn8m2tb

The incident has been resolved, but please let me know if you have any further issues with this 🙏
